src/Menu.cpp

```diff
@@ -1602,10 +1794,26 @@ void Menu::DrawPerformanceOverlaySettings()
 
 			// FPS update interval slider - Make this slider affect all FPS and frametime displays
 			ImGui::SliderFloat("Update Interval", &settings.PerfOverlay.UpdateInterval, 0.001f, 2.0f, "%.2f seconds");
+			ImGui::SameLine();
```

Comment:
I thought I had said this was inconsistent with other menus and should be dropped.